CHANGELOG.md

```diff
@@ -118,14 +118,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     - The `async-operations` endpoint will now omit `output` instead
       of returning `none` when the operation did not return output.
   - **CUMULUS-2167**
-    - Change Postgres Schema definition for Files to remove `filename` and `name` and only support `file_name`.
-    - Change Postgres Schema definition for Files to remove `size` to only support `file_size`.
-    - Change `PostgresFile` to remove duplicate fields `filename` and `name` and rename `size` to `file_size`.
+    - Changed Postgres Schema definition for Files to remove `filename` and `name` and only support `file_name`.
+    - Changed Postgres Schema definition for Files to remove `size` to only support `file_size`.
+    - Changed `PostgresFile` to remove duplicate fields `filename` and `name` and rename `size` to `file_size`.
   - **CUMULUS-2305**
     - Changed `DELETE /pdrs/{pdrname}` API behavior to also delete record from Postgres database.
   - **CUMULUS-2309**
     - Changed `DELETE /granules/{granuleName}` API behavior to also delete record from Postgres database.
     - Changed `Bulk operation BULK_GRANULE_DELETE` API behavior to also delete records from Postgres database.
+  - **CUMULUS-2385**
+    - Changed Postgres `File` schema to require `bucket` and `key` fields.
```

Comment:
I don't think this is technically a breaking change because RDS hasn't gone out, right?

Reply:
Yeah that sounds correct to me